api/src/main/resources/messages.properties

```diff
@@ -631,6 +631,8 @@ reporting.parameter.startDate=Start Date
 reporting.parameter.endDate=End Date
 reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
+reporting.parameter.attributeType=Attribute Type
+reporting.parameter.values=Value
```

Comment:
Thanks @mseaton in the light of your previous comments can we have the user enter a comma separated value for the input field so that the user can enter values like Germany, Nigeria, Brazil for an attribute like Citizenship. Also as regards the second comment, do we need to put that into consideration for this particular form cos I think all the values for the attributes listed only take values of type String